Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/PrefabProcessorContext.cpp

```diff
@@ -247,4 +247,16 @@ namespace AzToolsFramework::Prefab::PrefabConversionUtils
         return loadBehavior == EntityAliasSpawnableLoadBehavior::DependentLoad ? AZ::Data::AssetLoadBehavior::PreLoad
                                                                                : AZ::Data::AssetLoadBehavior::NoLoad;
     }
+
+    const AzToolsFramework::Prefab::InstanceAlias& PrefabProcessorContext::FindOrCreateCachedInstanceAliasForPrefab(const AZStd::string& prefabName)
+    {
+        auto it = m_cachedPrefabInstanceAliases.find(prefabName);
+        if (it != m_cachedPrefabInstanceAliases.end())
+        {
+            return it->second;
+        }
+
+        m_cachedPrefabInstanceAliases.insert({{prefabName, AzToolsFramework::Prefab::Instance::GenerateEntityAlias()}});
+        return FindOrCreateCachedInstanceAliasForPrefab(prefabName);
+    }
```

Comment:
Ok, good to know. 
Not sure how that `EntityAliasStore` can help here since all we need is `map<string, string>`. 
Probably better keep it simple rather than extending `EntityAliasStore` to have more variants that are not entity-level?